src/Tests/dotnet-new.Tests/DotnetNewUninstallTests.cs

```diff
@@ -198,7 +198,7 @@ public void CanUninstall_NuGet(string commandName)
                 .NotHaveStdErr()
                 .And.HaveStdOutContaining($"Currently installed items:{Environment.NewLine}(No Items)");
 
-            Assert.False(File.Exists(Path.Combine(home, "packages", "Microsoft.DotNet.Web.ProjectTemplates.5.0.5.0.0.nupkg")));
+            Assert.False(File.Exists(Path.Combine(home, ".templateengine", "packages", "Microsoft.DotNet.Web.ProjectTemplates.5.0.5.0.0.nupkg")));
```

Comment:
`Assert.False(File.Exists(Path.Combine(home, "packages", "Microsoft.DotNet.Web.ProjectTemplates.5.0.5.0.0.nupkg")));` is correct. 

Since we are using custom settings location, `".templateengine"` won't be appended